syntax = "proto3";

package telemetry;

message TelemetryMessage {
  uint32 sensor_id     = 1;
  double value         = 2;
  string unit          = 3;
  uint64 timestamp_ns  = 4;
  uint64 sequence      = 5;
}
